### PR TITLE
Documentation for filter convention

### PR DESCRIPTION
**Pull request by PascalSenn**
Adds documentaion for filter convention

### CODE REVIEW

File: website/src/components/layout.css [hunk 1]
```diff
@@ -251,7 +251,7 @@ h2 {
     Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
```
Comment (rstaib) [inline website/src/components/layout.css:251]: This file does not exist anymore. Please use `components/misc/global-style` instead.

File: website/src/docs/docs.json [hunk 2]
```diff
@@ -4,37 +4,110 @@
...
-    "items": [{ "path": "index", "title": "Introduction" }]
+    "items": [
+      {
+        "path": "index",
+        "title": "Introduction"
+      }
+    ]
   }
-]
+]
```
Comment (rstaib) [inline website/src/docs/docs.json:113]: Please install `prettier` and let it decide how to format things.

File: website/src/components/prism-highlight-line.css [hunk 3]
```diff
@@ -0,0 +1,10 @@
+.gatsby-highlight-code-line {
```
Comment (rstaib) [inline website/src/components/prism-highlight-line.css:1]: Please use `components/misc/global-style` instead.

File: website/gatsby-browser.js [hunk 4]
```diff
@@ -1 +1,2 @@
 import "prismjs/themes/prism-tomorrow.css";
+import "./src/components/prism-highlight-line.css";
```
Comment (rstaib) [inline website/gatsby-browser.js:2]: Remove.

File: website/src/docs/docs.json (hunk 2 above)
Comment (PascalSenn) [reply website/src/docs/docs.json:113]: interesting 
image

Comment (rstaib) [reply website/src/docs/docs.json:113]: Is it prettified? Sometimes it's not even if you see this flag :-D

Comment (rstaib) [reply website/src/docs/docs.json:113]: I will a git pre-hook for commits which will prettify everything to ensure it's pretty :-D

File: website/src/docs/hotchocolate/filtering/bool-filter.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
+### Boolean Filter
```
Comment (rstaib) [inline website/src/docs/hotchocolate/filtering/bool-filter.md:1]: Frontmatter missing...

```
---
title: "Here goes the headline..."
---
```

And do not use single `#` for subtitles; start with `##`.

File: website/src/docs/hotchocolate/filtering/comparable-filter.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
+### Comparable Filter
```
Comment (rstaib) [inline website/src/docs/hotchocolate/filtering/comparable-filter.md:1]: Frontmatter missing...

```
---
title: "Here goes the headline..."
---
```

And do not use single `#` for subtitles; start with `##`.

File: website/src/docs/hotchocolate/filtering/filter-and-operations.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
+# Filter & Operations Kinds
```
Comment (rstaib) [inline website/src/docs/hotchocolate/filtering/filter-and-operations.md:1]: Frontmatter missing...

```
---
title: "Here goes the headline..."
---
```

And do not use single `#` for subtitles; start with `##`.

File: website/src/docs/hotchocolate/filtering/filter-customization.md [hunk 8]
```diff
@@ -0,0 +1,424 @@
+# Customizing Filter
```
Comment (rstaib) [inline website/src/docs/hotchocolate/filtering/filter-customization.md:1]: Frontmatter missing...

```
---
title: "Here goes the headline..."
---
```

And do not use single `#` for subtitles; start with `##`.

File: website/src/docs/hotchocolate/filtering/filter-naming-convention.md [hunk 9]
```diff
@@ -0,0 +1,158 @@
+# Naming Conventions
```
Comment (rstaib) [inline website/src/docs/hotchocolate/filtering/filter-naming-convention.md:1]: Frontmatter missing...

```
---
title: "Here goes the headline..."
---
```

And do not use single `#` for subtitles; start with `##`.

File: website/src/docs/hotchocolate/filtering/list-filter.md [hunk 10]
```diff
@@ -0,0 +1,125 @@
+### List Filter
```
Comment (rstaib) [inline website/src/docs/hotchocolate/filtering/list-filter.md:1]: Frontmatter missing...

```
---
title: "Here goes the headline..."
---
```

And do not use single `#` for subtitles; start with `##`.

File: website/src/docs/hotchocolate/filtering/getting-started.md [hunk 11]
```diff
@@ -0,0 +1,198 @@
+---
+id: filters
```
Comment (rstaib) [inline website/src/docs/hotchocolate/filtering/getting-started.md:2]: Is `id` used?

File: website/src/docs/hotchocolate/filtering/object-filter.md [hunk 12]
```diff
@@ -0,0 +1,127 @@
+### Object Filter
```
Comment (rstaib) [inline website/src/docs/hotchocolate/filtering/object-filter.md:1]: Frontmatter missing...

```
---
title: "Here goes the headline..."
---
```

And do not use single `#` for subtitles; start with `##`.

File: website/src/docs/hotchocolate/filtering/string-filter.md [hunk 13]
```diff
@@ -0,0 +1,73 @@
+### String Filter
```
Comment (rstaib) [inline website/src/docs/hotchocolate/filtering/string-filter.md:1]: Frontmatter missing...

```
---
title: "Here goes the headline..."
---
```

And do not use single `#` for subtitles; start with `##`.

**Pull request merged**